### PR TITLE
logutil: add testcase for SlowQueryLogger.MaxDays/MaxSize/MaxBackups

### LINKED ISSUE #30315: add testcase for SlowQueryLogger.MaxDays/MaxSize/MaxBackups
## Enhancement
Add a testcase to cover https://github.com/pingcap/tidb/issues/25716 , in case we break it again.

### PR DESCRIPTION
**Pull request by guo-shaoge**
Signed-off-by: guo-shaoge <shaoge1994@163.com>

### What problem does this PR solve?

Issue Number: close #30315

Problem Summary: Add a testcase to cover #25716 , in case we break it again.

### Check List

Tests 

- [ ] Unit test
- [x] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by jyz0309 — APPROVED** [review]
LGTM

**Review by XuHuaiyu — APPROVED** [review]
LGTM

File: util/logutil/slow_query_logger.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ import (
 
 var _pool = buffer.NewPool()
 
-func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, error) {
+func newSlowQueryLogger(cfg *LogConfig) (*zap.Logger, *log.ZapProperties, *log.Config, error) {
 
 	// copy the global log config to slow log config
 	// if the filename of slow log config is empty, slow log will behave the same as global log.
```
Comment (wjhuang2016) [inline util/logutil/slow_query_logger.go:33]: How about extracting these codes into a function and testing it?

**Review by wshwsh12 — APPROVED** [review]

**Review by wjhuang2016 — APPROVED** [review]

**Pull request merged**